app/src/App.tsx

```diff
@@ -1,10 +1,15 @@
 import { NavigationContainer } from '@react-navigation/native';
 import React from 'react';
 import { Provider } from 'react-redux';
-import { store } from './store/app';
+import { createStore, logActionType } from './store/app';
 import StackNavigator from './screens/StackNavigator';
+import { bootstrapApp } from './store/bootstrap';
+
+const store = createStore(logActionType);
 
 const App = () => {
+  React.useEffect(() => store.dispatch(bootstrapApp()).abort, []);
```

Comment:
Redux toolkit async action creators automatically put an `AbortController` in the returned `Promise` when dispatching them. Reading up on the Redux toolkit docs recommends this so the `bootstrapApp` action stops dispatching further actions if `AbortController` is used to abort the thunk. The return value of `useEffect` is invoked when the component is dismounted, which in this case should really never happen since it's the root component but just in case.

Reply:
Can you put a comment above this line that explains it? It's not immediately clear.